rmldatatfbbox/rmldatatfbbox/core.py

```diff
@@ -5,77 +5,158 @@
-@click.command(help='TensorFlow Object Detection with bounding boxes.')
+@click.group(help='Dataset creation with bounding box info')
 @click.pass_context
 def tf_bbox(ctx):
     pass
+
+@tf_bbox.command(help='Create a dataset.')
+@pass_create
+@click.pass_context
+def create(ctx: click.Context, create: CreateInput):
+
+    cli_spinner("Importing TensorFlow...", _import_od)
+
+    config = create.config.get("plugin")
+
+    # Optional Information from interface (just contains plugin)
+    # metadata = create.plugin_metadata
+
+    # set up TF verbosity
+    if config.get('verbose'):
+        tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.INFO)
+    else:
+        tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.FATAL)
     
-# @tf_bbox.command(help='Create a dataset.')
-# def create(ctx):
-#     click.echo("Dataset gets created here.")
-
-# # stdout redirection found at https://codingdose.info/2018/03/22/supress-print-output-in-python/
-# def _import_od():
-#     """ Imports the necessary libraries for object detection training.
-#     Used to avoid importing them at the top of the file where they get imported
-#     on every ravenML command call, even those not to this plugin.
+    # set base directory for dataset 
+    base_dir = create.dataset_path
     
-#     Also suppresses warning outputs from the TF OD API.
-#     """
-#     # create a text trap and redirect stdout
-#     # to suppress printed warnings from object detection and tf
-#     text_trap = io.StringIO()
-#     sys.stdout = text_trap
-#     sys.stderr = text_trap
+    # Transformation functionality is not currently supported, thus the empty array
+    transform_metadata = []
+
+    # Variable type validation is currently not supported
+    dataset_name = config.get("dataset_name") if config.get("dataset_name") else user_input(message="What would you like to name this dataset?")
     
-#     # Calls to _dynamic_import below map to the following standard imports:
-#     #
-#     # import tensorflow as tf
-#     # from object_detection import model_hparams
-#     # from object_detection import model_lib
-#     _dynamic_import('tensorflow', 'tf')
-#     _dynamic_import('object_detection.model_hparams', 'model_hparams')
-#     _dynamic_import('object_detection.model_lib', 'model_lib')
-#     _dynamic_import('object_detection.exporter', 'exporter')
-#     _dynamic_import('object_detection.protos', 'pipeline_pb2', asfunction=True)
+    k_folds_specified = config.get("kfolds") if config.get("kfolds") else user_input(message="How many folds would you like the dataset to have?",default="5")
+
+    test_percent = config.get("test_percent") if config.get("test_percent") else .2
     
-#     # now restore stdout function
-#     sys.stdout = sys.__stdout__
-#     sys.stderr = sys.__stderr__
+    imageset_data = create.config.get("metadata").get("imageset_data")
+    labeled_images, label_to_int_dict = construct_all(imageset_data.get("image_ids"), imageset_data.get("temp_dir"))
```

Comment:
```suggestion
    imageset_data = create.config.["metadata"]["imageset_data"]
    labeled_images, label_to_int_dict = construct_all(imageset_data["image_ids"], imageset_data["temp_dir"])
```
nit: no need to use `.get` unless you specifically want it to default to `None` (same with other instances of this pattern below)